include/arch/arm/aarch64/cpu.h

```diff
@@ -179,4 +179,7 @@
 
 #define GET_MPIDR()		read_sysreg(mpidr_el1)
 #define MPIDR_TO_CORE(mpidr)	MPIDR_AFFLVL(mpidr, 0)
+
+#define IS_BOOTING_CORE()	(!MPIDR_TO_CORE(GET_MPIDR()))
```

Comment:
The macro name `IS_PRIMARY_CORE` is more accurate IMO.

Reply:
Done